backend/src/openarchiefbeheer/destruction/api/permissions.py

```diff
@@ -94,3 +94,17 @@ def has_object_permission(self, request, view, destruction_list):
             request.user == destruction_list.author
             and destruction_list.status == ListStatus.new
         )
+
+
+class CanAbortDestruction(permissions.BasePermission):
+    message = _("You are not allowed to stop the planned destruction of the list.")
```

Comment:
Maybe we should discuss whether it's more fail save if everyone (thats logged  in) can abort the destruction.